src/pages/Vaults/Vault/components/VaultInfo/VaultInfo.tsx

```diff
@@ -54,24 +54,28 @@ const VaultInfo = ({
   ...props
 }: VaultInfoProps): JSX.Element => {
   const [isModalOpen, setModalOpen] = useState(false);
-  const { children: definition, status } = getVaultStatus(vaultStatus);
-
-  const headlineItems: DlProps['listItems'] = [
-    { term: 'Vault ID', definition: shortAddress(vaultAddress) },
-    {
-      term: 'Vault Status',
-      definition,
-      render: (children: ReactNode) => <StatusTag status={status}>{children}</StatusTag>
-    }
-  ];
-
-  if (vaultDisplayName) {
-    headlineItems.push({ term: 'Identity', definition: vaultDisplayName });
-  }
+  const { children: vaultStatusDefinition, status } = getVaultStatus(vaultStatus);
 
   return (
     <StyledWrapper variant='bordered' {...props}>
-      <StyledDl listItems={headlineItems} />
+      <StyledDl>
```

Comment:
I'm not sure about this - our approach has been to render out complete components, and pass in an array/object to render the subcomponents. This pulls the individual elements out into separate components which is something I was trying to avoid. Is there a reason it has to be done like this?